src/Features/Core/Portable/Completion/CompletionList.cs

```diff
@@ -100,25 +114,25 @@ public sealed class CompletionList
 
         private CompletionList With(
             Optional<TextSpan> span = default,
-            Optional<ImmutableArray<CompletionItem>> items = default,
+            Optional<IReadOnlyList<CompletionItem>> items = default,
             Optional<CompletionRules> rules = default,
             Optional<CompletionItem> suggestionModeItem = default)
         {
             var newSpan = span.HasValue ? span.Value : Span;
-            var newItems = items.HasValue ? items.Value : Items;
+            var newItems = items.HasValue ? items.Value : ItemsInternal;
             var newRules = rules.HasValue ? rules.Value : Rules;
             var newSuggestionModeItem = suggestionModeItem.HasValue ? suggestionModeItem.Value : SuggestionModeItem;
 
             if (newSpan == Span &&
-                newItems == Items &&
+                newItems == ItemsInternal &&
                 newRules == Rules &&
                 newSuggestionModeItem == SuggestionModeItem)
             {
                 return this;
             }
             else
             {
-                return Create(newSpan, newItems, newRules, newSuggestionModeItem);
+                return Create(newSpan, newItems, newRules, newSuggestionModeItem, isExclusive: false);
```

Comment:
why `isExclusive: false`  here instead of `_isExclusive`?

Reply:
This was the previous behavior, can't remember why though. Will take a closer look later, but not gonna change in this PR.

Reply:
gotcha.  it is concerning.  but i agree it matches from before.  can you file issue?